### PR TITLE
E2E script needs to exit on errors

### PR DESCRIPTION
**Pull request by bparees**
## Description

Make sure we exit on any failure such as pulling down the RAG content or extracting it

## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
looks good

**Comment by bparees** [comment]
@vishnuchalla you were right, i should have done this in the first place :)

File: scripts/test-e2e-standalone.sh [hunk 1]
```diff
@@ -50,9 +50,13 @@ function finish() {
     echo Exit trap: killing OLS server
     kill %1
     rm -rf "$TMPDIR"
+    rm -rf "RAG_TMP_DIR"
```
Comment (onmete) [inline scripts/test-e2e-standalone.sh:53]: $RAG_TMP_DIR ?

Comment (bparees) [reply scripts/test-e2e-standalone.sh:53]: if you look at the top of the script it creates two tmpdirs for holding content, this ensures we clean both of them up.

Comment (onmete) [reply scripts/test-e2e-standalone.sh:53]: I mean isn't `$` missing there?

Comment (bparees) [reply scripts/test-e2e-standalone.sh]: ah, yes, thanks!

**Pull request merged**